sdcm/tester.py

```diff
@@ -1287,20 +1282,26 @@ def get_cluster_k8s_local_kind_cluster(self):
             n_nodes=self.params.get("k8s_n_scylla_pods_per_cluster") or self.params.get("n_db_nodes"),
             params=self.params,
             node_pool=scylla_pool,
+            add_nodes=False,
         )
 
         if self.params.get('k8s_deploy_monitoring'):
             self.k8s_cluster.deploy_monitoring_cluster(
                 is_manager_deployed=self.params.get('use_mgmt')
             )
         if self.params.get("n_loaders"):
-            self.loaders = cluster_docker.LoaderSetDocker(
-                **container_node_params,
-                n_nodes=self.params.get("n_loaders"),
+            self.loaders = cluster_k8s.LoaderPodCluster(
+                k8s_cluster=self.k8s_cluster,
+                loader_cluster_name=self.params.get("k8s_loader_cluster_name"),
                 user_prefix=self.params.get("user_prefix"),
+                n_nodes=self.params.get("k8s_n_loader_pods_per_cluster") or self.params.get("n_loaders"),
                 params=self.params,
+                node_pool=scylla_pool,
```

Comment:
It must not be on the `scylla-pool`.
Need to create `loader-pool` and reuse it here.

Thanks for the switching it to the K8S-based laoder.

Reply:
changed